apolloschurchapp/src/tabs/home/Features/HorizontalCardListFeature/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { PureComponent } from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import {
+  H2,
+  H5,
+  HorizontalTileFeed,
+  PaddedView,
+  styled,
+  TouchableScale,
+  withIsLoading,
+} from '@apollosproject/ui-kit';
+import { horizontalContentCardComponentMapper } from '@apollosproject/ui-connected';
+
+const Title = styled(
+  ({ theme }) => ({
+    color: theme.colors.text.tertiary,
+  }),
+  'HorizontalCardListFeature.Title'
+)(H5);
+
+const Subtitle = styled({}, 'HorizontalCardListFeature.Subtitle')(H2);
+
+const Header = styled(({ theme }) => ({
+  paddingTop: theme.sizing.baseUnit * 3,
+  paddingBottom: theme.sizing.baseUnit * 0.5,
+}))(PaddedView);
+
+class HorizontalCardListFeature extends PureComponent {
+  loadingStateObject = {
+    id: 'fakeId0',
+    isLoading: true,
```

Comment:
weren't you and @richarddubay just talking about this?!

Reply:
We were. That's one  of the reasons I was so familiar with what he was trying to do. The code was very front of mind. I paired with him for a bit and got a few take aways we can discuss later on how we can improve some of the code he was working with.